src/Util/ReflectionClassRecursiveIterator.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Util;
+
+/**
+ * Gets reflection classes for php files in the given directories.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ *
+ * @internal
+ */
+final class ReflectionClassRecursiveIterator
```

Comment:
You should add a private constructor.

Reply:
Really necessary? This class is already marked as `@internal`.

Reply:
has already be fixed anyway